src/transformers/models/auto/configuration_auto.py

```diff
@@ -548,6 +550,7 @@
         ("lilt", "LiLT"),
         ("llama", "LLaMA"),
         ("llama2", "Llama2"),
+        ("llava", "Llava"),
```

Comment:
```suggestion
        ("llava", "LLaVa"),
```
Make sure to run `make fix-copies` after this